Docs/source/running_cpp/parameters.rst

```diff
@@ -244,24 +246,47 @@ Math parser and user-defined constants
 --------------------------------------
 
 WarpX provides a math parser that reads expressions in the input file.
-It can be used to define the plasma density profile, the plasma momentum
-distribution or the laser field (see below `Particle initialization` and
-`Laser initialization`).
+It can be used in all input parameters that consist in one real number.
 
-The parser reads python-style expressions between double quotes, for instance
-``"a0*x**2 * (1-y*1.e2) * (x>0)"`` is a valid expression where ``a0`` is a
-user-defined constant and ``x`` and ``y`` are variables. The names are case sensitive. The factor
-``(x>0)`` is `1` where `x>0` and `0` where `x<=0`. It allows the user to
-define functions by intervals. User-defined constants can be used in parsed
-functions only (i.e., ``density_function(x,y,z)`` and ``field_function(X,Y,t)``,
-see below). User-defined constants can contain only letters, numbers and the character ``_``.
+WarpX constants
+###############
+
+WarpX provides a few pre-defined constants, that can be used for any parameter that consists in one real number.
+
+======== ===================
+q_e      elementary charge
+m_e      electron mass
+m_p      proton mass
+epsilon0 vacuum permittivity
+clight   speed of light
```

Comment:
Do we have a preference for `clight` over `c0`/`c_0`? :)
I slightly prefer a shorter name.

Reply:
I don't mind, I just tried to avoid too short names now that we decided to have no prefix. Would you rather have c0?

Reply:
Let's do a quick question as above: @RemiLehe @jlvay voted above
:+1: `c_0`
:rocket: `c0`
:tada: `clight`

Reply:
I am fine with `clight` but `c0` is definitely shorter and thus nicer to type in expressions. In that spirit, what about `eps0` instead of `epsilon0`?